src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/takecard/events/CompleteDragEvent.kt

```diff
@@ -0,0 +1,17 @@
+package org.wycliffeassociates.otter.jvm.app.widgets.takecard.events
+
+import javafx.event.Event
+import javafx.event.EventType
+import javafx.scene.input.MouseEvent
+import org.wycliffeassociates.otter.common.data.workbook.Take
+
+class CompleteDragEvent(
+    type: EventType<CompleteDragEvent>,
```

Comment:
Do you need to accept type in the constructor?  Can you just use the companion value?